backend/src/vscode-youi-events.ts

```diff
@@ -7,11 +7,13 @@ import { GeneratorFilter, GeneratorType } from './filter';
 export class VSCodeYouiEvents implements YouiEvents {
     private webviewPanel: vscode.WebviewPanel;
     private genFilter: GeneratorFilter;
+    private messages: any;
     private resolveFunc: any;
 
-    constructor(rpc : IRpc, webviewPanel: vscode.WebviewPanel, genFilter: GeneratorFilter) {
+    constructor(rpc : IRpc, webviewPanel: vscode.WebviewPanel, genFilter: GeneratorFilter, messages: any) {
```

Comment:
genFilter: GeneratorFilter, messages: any
could bepassed in 1 obj {}

Reply:
I will do it in my next PR
Thank you